frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverview.tsx

```diff
@@ -33,7 +33,7 @@ type MonitoringOverviewProps = {
 const MonitoringOverview: React.FC<MonitoringOverviewProps> = (props) => {
   const { resource, pods, resourceEvents } = props;
   const [expanded, setExpanded] = React.useState(['metrics']);
-
+  const canAccessMonitoring = !!window.SERVER_FLAGS.prometheusBaseURL;
```

Comment:
Should this be `window.SERVER_FLAGS. prometheusTenancyBaseURL` ?

Reply:
yes.

Reply:
I don't see what the changes in this file are achieving. Could you please explain the intention here?

Reply:
Undo the changes. As ```prometheusTenancyBaseURL``` is always available I removed the checks for it that I added.